logstash-core/src/test/java/org/logstash/ackedqueue/QueueTest.java

```diff
@@ -994,4 +1008,12 @@ public void maximizeBatch() throws IOException, InterruptedException, ExecutionE
             assertThat(b.size(), is(2));
         }
     }
+
+    @Test(expected = IOException.class)
+    public void throwsWhenNotEnoughDiskFree() throws Exception {
+        Settings settings = SettingsImpl.builder(TestSettings.persistedQueueSettings(100, dataPath)).queueMaxBytes(Long.MAX_VALUE).build();
```

Comment:
nitpick: the relevant bits of this test are way off to the right; a minor change in indentation allows us to bring them to the forefront and add useful commentary:

~~~diff
-        Settings settings = SettingsImpl.builder(TestSettings.persistedQueueSettings(100, dataPath)).queueMaxBytes(Long.MAX_VALUE).build();
+        Settings settings = SettingsImpl.builder(TestSettings.persistedQueueSettings(100, dataPath))
+                                        .queueMaxBytes(Long.MAX_VALUE) // ask for far too much disk
+                                        .build();
~~~

Reply:
+1